people.qmd

```diff
@@ -0,0 +1,131 @@
+---
+title: "People"
+comments: false
+page-layout: full
+---
+
+## Principal Investigator
+
+::: {.grid}
+::: {.g-col-12 .g-col-md-12}
+
+<div class="card mb-3">
+<div class="card-body">
+<img src="assets/images/js.png"/>
+<h3 class="card-title">JooYoung Seo</h3>
```

Comment:
Could you please upload our headshot assets there, including yours and Aziz's? Mine can be found at my faculty site.